packages/cfd/src/Containers/dynamic-leverage/dynamic-leverage-market-card.tsx

```diff
@@ -0,0 +1,89 @@
+import { TDynamicLeverageMarketCardProps } from 'Containers/props.types';
+
+export const DynamicLeverageMarketCard = ({
+    market,
+    leverage,
+    market_example,
+    dynamicLeverages,
+}: TDynamicLeverageMarketCardProps) => (
+    <div className='dynamic-leverage-modal__market'>
+        <div className='dynamic-leverage-modal__market-title'>
+            <Text data-testid='market_title' size='xs' weight='bolder' align='center'>
+                {localize(market)}
+            </Text>
+            {market_example ? (
+                <Text
+                    data-testid='example_title'
+                    size='xxxs'
+                    align='center'
+                    className='dynamic-leverage-modal__market-example'
+                >
+                    {localize(market_example)}
+                </Text>
+            ) : null}
+            <Text data-testid='leverage_title' size='xxs' color='red' align='center'>
+                {localize(leverage)}
+            </Text>
+        </div>
+        <Table className='dynamic-leverage-modal__market-table'>
+            <Table.Header>
+                <Table.Row className='dynamic-leverage-modal__market-table-header-row'>
+                    <Table.Head>
+                        <div className='dynamic-leverage-modal__market-table-header'>
+                            <Text size='xs' align='center' weight='bold'>
+                                {localize('From')}
+                            </Text>
+                            <Text size='xxs' align='center'>
+                                {localize('(lots)')}
+                            </Text>
+                        </div>
+                    </Table.Head>
+                    <Table.Head>
+                        <div className='dynamic-leverage-modal__market-table-header'>
+                            <Text size='xs' align='center' weight='bold'>
+                                {localize('to')}
+                            </Text>
+                            <Text size='xxs' align='center'>
+                                {localize('(lots)')}
+                            </Text>
+                        </div>
+                    </Table.Head>
+                    <Table.Head>
+                        <div className='dynamic-leverage-modal__market-table-header'>
+                            <Text size='xs' align='center' weight='bold'>
+                                {localize('Leverage')}
+                            </Text>
+                            <Text size='xxs' align='center'>
+                                {localize('(1:x)')}
+                            </Text>
+                        </div>
+                    </Table.Head>
+                </Table.Row>
+            </Table.Header>
+            <Table.Body>
+                {dynamicLeverages.map((x, index) => (
+                    <Table.Row key={index} className='dynamic-leverage-modal__market-table-row'>
```

Comment:
```suggestion
                    <Table.Row key={index} className='dynamic-leverage-modal__market-table-row'>
```
Using index as key is not recommended, could cause issues of dynamic leverages array changes

Reply:
```suggestion
                    <Table.Row key={x.from} className='dynamic-leverage-modal__market-table-row'>
```
@arshad-rao-deriv i suspect `from` is different in every object, so we can use it instead of index.

Reply:
Updated.